src/common/utility.js

```diff
@@ -430,6 +430,18 @@ export const useCallbackRef = (callback) => {
 
 const eu_subdomain_countries = eu_countries.filter((country) => country !== 'gb')
 
+export const isEuDomain = () => {
+    if (isBrowser()) {
+        if (
+            window.location.hostname === 'eu.deriv.com' ||
+            window.location.hostname === 'staging-eu.deriv.com'
```

Comment:
I remember we already have an array containing these 2 domains. Can you help finding it and consolidate them in one exported array so everywhere in the codebase use that array. 

Then instead of if else we can use `[].includes()`